### PR TITLE
[sw, rom_ext_signer] Add minimal rustfmt configuration file

### PR DESCRIPTION
**Pull request by silvestrst**
This PR makes sure that lines over 80 characters are split.

Please note that toolchain removal commit is a duplicate of another PR, and one of them will have to be rebased dependent on which lands first.

### CODE REVIEW

**Comment by cfrantz** [comment]
Are all of the source file changes a result of applying the formatting rules?

**Comment by silvestrst** [comment]
> Are all of the source file changes a result of applying the formatting rules?

Yes, just `cargo fmt`

File: sw/host/rom_ext_image_tools/signer/rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+max_width = 80
```
Comment (imphil) [inline sw/host/rom_ext_image_tools/signer/rustfmt.toml:5]: Missing a trailing newline.

**Review by imphil — COMMENTED** [review]
Could you also open an issue (and/or attempt to address it directly) to add a rust lint check to CI?

**Comment by silvestrst** [comment]
> Could you also open an issue (and/or attempt to address it directly) to add a rust lint check to CI?

Can do both, thanks!

**Review by mcy — APPROVED** [review]
LG. Can you file the followup presubmit PR before merging this?

**Review by cfrantz — APPROVED** [review]

File: sw/host/rom_ext_image_tools/signer/rustfmt.toml (hunk 1 above)
Comment (silvestrst) [reply sw/host/rom_ext_image_tools/signer/rustfmt.toml]: Done.

**Pull request merged**